src/components/wrappers/TogglingBlock.vue

```diff
@@ -0,0 +1,178 @@
+<script>
+import PopUp from '@/components/wrappers/PopUp.vue';
+import { mapGetters } from 'vuex';
+
+export default {
+  components: {
+    PopUp,
+  },
+  props: {
+    forceShow: {
+      type: Boolean,
+      default: false,
+    },
+    title: {
+      type: String,
+      default: '',
+    },
+    hintText: {
+      type: String,
+      default: '',
+    },
+    caps: {
+      type: Boolean,
+      default: false,
+    },
+    bordered: {
+      type: Boolean,
+      default: true,
+    },
+  },
+  data() {
+    return {
+      isOpened: false,
+    };
+  },
+  computed: {
+    ...mapGetters('settings', [
+      'isItemFormInSidebar',
+    ]),
+  },
+  mounted() {
+    if (this.forceShow) {
+      this.isOpened = true;
+    }
+  },
+  methods: {
+    toggleBlockVisibility() {
+      this.isOpened = !this.isOpened;
+    },
+  },
+};
+</script>
+
+<template>
+  <div
+    class="toggling-block"
+    :class="{ bordered }"
+  >
+    <div
+      class="button-area"
+      :class="{ opened: isOpened }"
+      @click="toggleBlockVisibility"
+    >
+      <div class="title-section">
+        <div
+          class="title"
+          :class="{ caps }"
+        >
+          {{ title }}
+        </div>
+        <div
+          v-if="hintText"
+          class="hint-area"
+        >
+          <PopUp 
+            button-style-type="hint"
+            stop-propagation
+            informational
```

Comment:
informational and button-style-type isn't doing in the same field of responsibility?

Reply:
no, informational are currently used both in "hint" and "info" button

Reply:
ok, let it be in the task for checking consistency